server/views/pages/offenceCodeProtectedCharacteristics.njk

```diff
@@ -0,0 +1,106 @@
+{% extends "../partials/layout.njk" %}
+{% from "../partials/breadCrumb.njk" import breadcrumb %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set title = "Offence details" %}
+
+{% block beforeContent %}
+  {{ breadcrumb() }}
+{% endblock %}
+
+{% block content %}
+  {% if errors | length %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { "data-qa": "error-summary" }
+      }) }}
+  {% endif %}
+
+{% set items =
+[
+    {
+      value: ProtectedCharacteristicsTypes.AGE,
+      text: "Age"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.DISABILITY,
+      text: "Disability"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.GENDER_REASSIGN,
+      text: "Gender reassignment"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.MARRIAGE_AND_CP,
+      text: "Marriage and civil partnership"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.PREGNANCY_AND_MAT,
+      text: "Pregnancy and maternity"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.RACE,
+      text: "Race"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.RELIGION,
+      text: "Reglion or belief"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.SEX,
+      text: "Sex"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.SEX_ORIENTATION,
+      text: "Sexual orientation"
+    }
+  ] %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+  <form method="POST" novalidate="novalidate" class="govuk-!-margin-top-5" data-qa="offence-code-protected-characteristics-form">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+    <input type="hidden" name="protectedCharacteristicPage" value="true"/>
```

Comment:
This is a flag so we can tell which njk we're rendering if there's a validation error so we know what to check for